src/command.ts

```diff
@@ -36,21 +36,33 @@ function takeScreenshot(subject: any, name: string, screenshotOptions: any): voi
     objToOperateOn = cy
   }
 
+  let screenshotPath: string
   // eslint-disable-next-line promise/catch-or-return
-  objToOperateOn.screenshot(name, screenshotOptions).then(() => {
-    return null
-  })
+  objToOperateOn
+    .screenshot(name, {
+      ...screenshotOptions,
+      onAfterScreenshot(_el: any, props: any) {
+        screenshotPath = props.path
```

Comment:
ok, so the main idea here is that we don't want/need to calculate the path using the screenshotFolder + specName + path.join stuff, just use this absolute path we anyway get in the onAfterScreenshot callback

Reply:
I agree , simple is better